pallets/liquid-staking/src/lib.rs

```diff
@@ -388,10 +374,7 @@ pub mod pallet {
 
             // calculate staking fee and add it to insurance pool
             let fees = Self::reserve_factor().mul_floor(amount);
-            InsurancePool::<T>::try_mutate(|b| -> DispatchResult {
-                *b = b.checked_add(fees).ok_or(ArithmeticError::Overflow)?;
-                Ok(())
-            })?;
+            T::XCM::update_insurance_pool(fees)?;
```

Comment:
insurance_pool shouldn't be moved into pallet-xcm-helper as it's liquid staking specific thing

Reply:
replied here https://github.com/parallel-finance/parallel/pull/1038#discussion_r770181244